code/modules/economy/account.dm

```diff
@@ -56,9 +58,13 @@
 	return FALSE
 
 /datum/bank_account/proc/payday(amt_of_paychecks, free = FALSE)
+	if(suspended)
+		bank_card_talk("ERROR: Payday aborted, the account holder had been moved to cryo storage.")
```

Comment:
"is in cryogenic storage"

Reply:
Should be general I think, so you can close someones bank account for fraud as an admin